Source/WebCore/loader/FrameLoader.h

```diff
@@ -552,6 +558,8 @@ class FrameLoader final : public CanMakeWeakPtr<FrameLoader> {
     bool m_doNotAbortNavigationAPI { false };
     RefPtr<HistoryItem> m_pendingNavigationAPIItem;
     uint64_t m_requiredCookiesVersion { 0 };
+
+    RefPtr<DocumentPrefetcher> m_documentPrefetcher;
```

Comment:
It seems like this should be a const Ref instead of a RefPtr because it never changes and is never null.